### DISCUSSION TITLE
Any plans to add a datepicker/calendarpicker

### DISCUSSION THREAD

**Discussion by Alek99** [chakra-ui/chakra-ui#7212] [General] [answered]
Any plans to add these components in the near future? They seem essential to a lot of projects so it would be nice to have in Chakra. I see most other libraries like Ant, Material, Bootstrap have this included. Thanks!

**Comment by ziyuez** [comment, marked as answer]
You might be looking for the `date` and `datetime` type of the [input component](https://chakra-ui.com/docs/components/input)!
<img width="828" alt="Screen Shot 2023-01-16 at 12 26 47 PM" src="https://user-images.githubusercontent.com/22438541/212760898-f6e0186e-05a9-4a41-a296-cb7c5eb71500.png">

**Reply by Alek99** [reply]
Thats what I needed thank missed this

**Reply by alifa20** [reply]
Thanks a lot @ziyuez 🙏

**Comment by ajomuch92** [comment]
I created a package with a custom date picker based on Chakra UI, see it [here](https://www.npmjs.com/package/simple-chakra-ui-datepicker)
